src/main/java/uk/gov/hmcts/reform/pip/rules/repository/InMemoryRepository.java

```diff
@@ -20,6 +36,25 @@
 
     private static final Integer AVAILABLE_PUBLICATION = 1;
 
+
+
+    public InMemoryRepository() throws IOException {
+
+        ObjectMapper om = new ObjectMapper();
+        DateFormat df = new SimpleDateFormat("dd/MM/yyyy");
+        om.setDateFormat(df);
+        File file = new File("src/main/java/uk/gov/hmcts/reform/pip/rules/repository/mocks/hearingsList.json");
+        Hearing[] list = om.readValue(file, Hearing[].class);
+        this.listHearings = Arrays.asList(list);
+
+        file = new File("src/main/java/uk/gov/hmcts/reform/pip/rules/repository/mocks/courtsAndHearingsCount.json");
```

Comment:
youre building 2 separate objects here, these should be split into their own builders

Reply:
split in 2 builders